packages/edit-post/src/components/text-editor/index.js

```diff
@@ -4,54 +4,43 @@
 import {
 	PostTextEditor,
 	PostTitle,
-	TextEditorGlobalKeyboardShortcuts,
 	store as editorStore,
+	TextEditorGlobalKeyboardShortcuts,
 } from '@wordpress/editor';
-import { Button } from '@wordpress/components';
-import { withDispatch, withSelect } from '@wordpress/data';
-import { __ } from '@wordpress/i18n';
-import { displayShortcut } from '@wordpress/keycodes';
-import { compose } from '@wordpress/compose';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { CodeEditorScreen } from '@wordpress/interface';
```

Comment:
Why CodeEditorScreen is part of the interface package? It seems very misplaced for me?

Reply:
I think sharing code just to share code is bad, that said, I think I may have misunderstood this component first, what is it for?

Reply:
CodeEditorScreen is a skeleton for the editor code editor mode. It renders a big element where one can nest areas relevant for code editing (normally the code editor and maybe the post title) and also renders a button to exit the code editor.

> I think sharing code just to share code is bad

In this case, we have a UI for editing posts that is exactly the same between post editor and site editor. This UI has lots of CSS, duplicating all of this seems bad, soon one will have to do a small design change and will only change in one place and forget to change it on the other.